plugins/utils/which-key.nix

```diff
@@ -559,26 +559,72 @@ lib.nixvim.neovim-plugin.mkNeovimPlugin config {
       '';
       visible = false;
     };
+
+    mappingsWithOpts = mkOption {
```

Comment:
Could also simply be named `mappings`, with a note that mappings can also be defined in `settings.spec`?

Reply:
Updated to use the mappings name, since upstream treats opts as a known optional parameter for mappings and recommends calling the `wk.add()` like this.